cmd/influx_tools/export/README.md

```diff
@@ -73,4 +73,99 @@ Seq #           ID              Start                           End
 
 A range can either be a single sequence number or an interval as shown previously.
 
-**Hint**: Include the `-print-only` option to display the plan and exit without exporting any data. 
\ No newline at end of file
+**Hint**: Include the `-print-only` option to display the plan and exit without exporting any data.
+
+Example usage for Enterprise
+----------------------------
+
+Note: This is only for fully replicated clusters. For non-fully-replicated clusters (RF < # of nodes),
+the current best method would be exporting in line protocol and re-importing after deleting the old shards.
+
+This example takes many small shards and collapses them to fewer large shards, which can be beneficial
+for TSI index performance.
+
+1) Create collapsedir containing export/import data from all shards:
+
+This can be done on a running backup of the cluster - `influx_tools` needs to be able
+to query the cluster metadata and the on-disk state for a data node. Ensure any environment
+variables and config file settings for the real `influxd` are reflected in the config file
+provided to `./influx_tools`.
+
+First do the dry run. The `-duration` must be an *integer multiple* of the shard group duration being
```

Comment:
The sequence of steps could probably be graphically differentiated better, e.g. with a numbered list or maybe headers. As is, it's a bit unclear if "do the dry run" is different from the sentence "On one of the nodes..." below